packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts

```diff
@@ -245,7 +246,7 @@ export class NgOptimizedImage implements OnInit, OnChanges, OnDestroy {
    * Image name will be processed by the image loader and the final URL will be applied as the `src`
    * property of the image.
    */
-  @Input({required: true}) ngSrc!: string;
+  @Input({required: true, transform: makeSafeValueString}) ngSrc!: string;
```

Comment:
I'd propose renaming the function a bit to make it more obvious what the transformation that we are making:
```suggestion
  @Input({required: true, transform: unwrapSafeUrl}) ngSrc!: string;
```

Reply:
Looks good. Done